lib/slack/index.js

```diff
@@ -43,28 +43,70 @@ module.exports = (robot) => {
   router.use('/actions', middleware.routeAction);
 
   // Format: '/actions:type:callback_id:action'
-  router.post(/actions:interactive_message:unfurl-\d+:unfurl-dismiss/, unfurls.showRichPreview.dismissPrompt);
-  router.post(/actions:interactive_message:unfurl-\d+:unfurl/, unfurls.showRichPreview.deliverUnfurl);
+  router.post(
```

Comment:
there seems to be some auto-formatting at play here? Could you undo these changes just to keep the diff clean? 😬